spec/requests/providers/capital_assessment_results_controller_spec.rb

```diff
@@ -46,6 +46,10 @@
           it "displays the correct result" do
             expect(unescaped_response_body).to include(I18n.t("ineligible.heading", name: applicant_name, scope: locale_scope))
```

Comment:
**nit** not you but could refactor as for other comments "I think it is clear and simpler and less coupled to simple expect the text/string that you expect"

Reply:
👍 have amended to use string instead of translation in spec expectation